Update knit code v2 (#60)
JetBrains/koog-docs: docs/custom-nodes.md

    // Custom logic
    input // Return the input as output (pass-through)
}

val myCustomNode by myCustomNode("node_name")
```
<!--- KNIT example-custom-nodes-03.kt -->

This creates a pass-through node that performs some custom logic but returns the input as the output without modification.

### Nodes with additional arguments

You can create nodes that accept arguments to customize their behavior:

<!--- INCLUDE
import ai.koog.agents.core.dsl.builder.AIAgentNodeDelegate
import ai.koog.agents.core.dsl.builder.AIAgentSubgraphBuilderBase
import ai.koog.agents.core.dsl.builder.strategy

typealias Input = String
typealias Output = String

val strategy = strategy<String, String>("strategy_name") {
-->
<!--- SUFFIX
}
-->
```kotlin
    fun AIAgentSubgraphBuilderBase<*, *>.myNodeWithArguments(
    name: String? = null,
    arg1: String,
    arg2: Int
): AIAgentNodeDelegate<Input, Output> = node(name) { input ->
    // Use arg1 and arg2 in your custom logic
    input // Return the input as the output
}

val myCustomNode by myNodeWithArguments("node_name", arg1 = "value1", arg2 = 42)
```
<!--- KNIT example-custom-nodes-04.kt -->

> ### Parametrized nodes

Reviewer: ```suggestion
### Parameterized nodes
```

> You can define input, output parameter on the nodes:

Reviewer: Maybe something like this would be better? 
```suggestion
You can define nodes with input and output parameters:
```
